src/js/controller/base.js

```diff
@@ -189,6 +189,14 @@ Base.prototype.updateSchedule = function(schedule, options) {
         schedule.set('isFocused', options.isFocused);
     }
 
+    if (options.location) {
+        schedule.set('location', options.location);
+    }
+
+    if (options.state) {
```

Comment:
Is this necessary for your issue? I agree this is a bug, but not related with `location`. What do you think?

Reply:
It's another not related property, but it was also not saved. So I added it in the PR.
Yes, I would need both for my requirements.

Reply:
Ok. Then I will add new property and description at `schedule.js`. It's just an information for you.
```js
/**
 * The model of calendar schedules.
 * @constructor
 * @mixes dirty
 * @mixes model
 */
function Schedule() {
...
    /**
     * location
     * @type {string}
     */
    this.location = '';

    /**
     * state
     * @type {string}
     */
    this.state = '';
...
}
```